docs/language/contracts.rst

```diff
@@ -79,10 +79,11 @@ _______________________________________________________
 When a new contract is created, the address for the new contract is a hash of the input
 (the constructor arguments) to the new contract. So, a contract cannot be created twice
 with the same input. This is why the salt is concatenated to the input. The salt is
-either a random value or it can be explicitly set using the ``{salt: 2}`` syntax. A
+either a random value or it can be explicitly set using the
+``{salt: hex"439d399ee3b5b0fae6c8d567a8cbfa22d59f8f2c5fe308fd0a92366c116e5f1a"}`` syntax. A
```

Comment:
When you use `either ... or ...`, the forms before and after the `or` must be the same. `a random value` is a noun phrase and `it can be explicitly set` is a verb clause.

I suggest writing `The salt can either be a random value or set explicitly using the syntax`.

Reply:
I've re-written this paragraph